### PR TITLE
chore: require protobuf depedency

### PR DESCRIPTION
**Pull request by bruuuuuuuce**

### CODE REVIEW

**Review by rishtigupta — APPROVED** [review]

**Review by pgautier404 — APPROVED** [review]

**Review by cprice404 — APPROVED** [review]
this lgtm, if @pgautier404 approves then let's ship it and cut a release!

**Comment by cprice404** [comment]
> this lgtm, if @pgautier404 approves then let's ship it and cut a release!

oh he already did! 🚢

File: composer.json [hunk 1]
```diff
@@ -39,10 +39,10 @@
         }
     },
     "require": {
-        "php": ">=7.4",
+        "ext-protobuf": "*",
         "ext-grpc": "*",
+        "php": ">=7.4",
         "firebase/php-jwt": "^6.3",
-        "google/protobuf": "3.22.3",
         "grpc/grpc": "1.52.0",
```
Comment (GrahamCampbell) [inline composer.json:46]: is it possible to bump this to 1.57.0?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (bruuuuuuuce) [reply composer.json:46]: Yea good call, I'll open a new pr with that change and make sure it gets into our next release

Comment (bruuuuuuuce) [reply composer.json:46]: https://github.com/momentohq/client-sdk-php/pull/180

**Pull request merged**